src/main/kotlin/com/manimdsl/frontend/AbstractSyntaxTree.kt

```diff
@@ -15,6 +15,9 @@ data class FunctionNode(
         val statements: List<StatementNode>
 ) : CodeNode(lineNumber)
 
+data class ParameterListNode(val parameters: List<ParameterNode>) : ASTNode()
+data class ParameterNode(val identifier: String, val type: Type) : ASTNode()
```

Comment:
Was this not here from a previous PR?

Reply:
It was in `Type.kt` so I moved it into `AbstractSyntaxTree.kt`

Reply:
I might have put them in the wrong file